runtime/common/src/test/kotlin/kotlinx/serialization/json/polymorphic/JsonNestedPolymorphismTest.kt

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2017-2019 JetBrains s.r.o. Use of this source code is governed by the Apache 2.0 license.
+ */
+
+package kotlinx.serialization.json.polymorphic
+
+import kotlinx.serialization.*
+import kotlinx.serialization.context.*
+import kotlinx.serialization.json.*
+import kotlin.test.*
+
+class JsonNestedPolymorphismTest : JsonTestBase() {
+
+    @BeforeTest
+    fun setUp() {
+        unquoted.install(object : SerialModule {
```

Comment:
you can use `installPolymorphicModule` extension on `SerialFormat` and also `.bind/.rebind` functionality